### ISSUE TITLE
Access to dataset

### ISSUE THREAD

**Issue by zzw-angry** [mtics/FedRAP#3]
Dear Author, thank you for your outstanding contribution to the intelligent federated recommendation system. Could you please let us know if there is a direct way to access the datasets used in the experiments?

**Comment by zzw-angry** [comment]
I have found it

**Comment by mtics** [comment]
No, you need to download the dataset with url provided in the appendix.
